napari/utils/colormaps/colormap_utils.py

```diff
@@ -622,7 +624,8 @@ def ensure_colormap(colormap: ValidColormapArg) -> Colormap:
                 and isinstance(colormap[0], str)
                 and isinstance(colormap[1], (VispyColormap, Colormap))
             ):
-                name, cmap = colormap
+                name = colormap[0]
+                cmap = colormap[1]
```

Comment:
For some reason `mypy` doesn't like the original unpacking, but it is fine this way round.

Reply:
Do you mean some local running mypy, not typing action?

Reply:
Yeah, I've based this PR off errors detected locally but I haven't enabled them on the CI, mainly because that will be much easier to do when https://github.com/napari/napari/pull/5825 is merged

Reply:
Ok. I checked `ValidColormapArg`, and it looks like mypy may have a problem with this.